common/keyValuePair.go

```diff
@@ -0,0 +1,8 @@
+package common
+
+// KeyValuePair is a tuple of (key, value)
+// TODO: Move this to elrond-go-core.
+type KeyValuePair struct {
```

Comment:
Thanks, missed that. The struct isn't exported, though. One option would be to define an interface for `KeyValueStorage` in `storer/interface.go`. Another option would be to convert `keyValStorage` to an exported plain struct, to always be passed by value (of course, it will still hold pointers to key & value).

Keep as it is? What would be best in this case?

Reply:
I think that you should keep this `KeyValuePair` (and move it to core), because the keyValStorage is pretty specialized for trie values, as the struct also contains the function ValueWithoutSuffix(). This way, keyValStorage can "extend" `KeyValuePair`